main.py

```diff
@@ -0,0 +1,17 @@
+from flask import Flask
+from flask_migrate import Migrate
+from src.extensions import db
+from src.controllers.faculty_controller import faculty_blueprint
+
+app = Flask(__name__)
```

Comment:
add requirements.txt file with all requirments that need to be installed for this project.